app/components/page_list_component/view.html.erb

```diff
@@ -32,7 +32,7 @@
             </dd>
           </div>
 
-          <% if FeatureService.enabled?(:basic_routing) %>
+          <% if @can_edit_page_routing %>
```

Comment:
Getting in here before @aliuk2012 does - suggest using attr_accessor in the view.rb to allow you to remove the `@`:
```suggestion
          <% if can_edit_page_routing %>
```

Reply:
You could potentially do the same with `@form` and `@pages` but if you want to keep this PR focused on the policy change I'm happy to do it in the `can_add_page_routing_conditions?` update PR